README.md

```diff
@@ -44,3 +44,17 @@ The curator workflow is as follows:
 - [ ] If a proposal is accepted, update both the label AND status to 'accepted', and when closing select 'Close as completed'
 - [ ] If a proposal is turned down, update both the label AND status to 'turned-down', and when closing select 'Close as not planned'
 - [ ] If a proposal is not completed or abandoned after a year, update both the label AND status to 'timed-out', and when closing select 'Close as not planned'
+
+### Request for Comment (RFC)
+
+The curator workflow is as follows:
+
+- [ ] Once a issue is made, update the 'Project' status to 'proposed' (right hand side bar, under 'new-<TYPE>-proposals')
+- [ ] Facilitate discussion on the the Issue thread, following the guidance [here](https://nf-co.re/docs/contributing/project_proposals).
+- [ ] Acceptance requires quorum from the nf-core core team, done via voting in comments with `/accept` or `/reject`
+- [ ] If a proposal is turned down, update both the label AND status to 'turned-down', and when closing select 'Close as not planned'
+- [ ] If a proposal is accepted, update both the label AND status to 'accepted', and ask the champion to write a formal RFC and begin initial development.
+- [ ] When the PR is opened the corresponding issue can be closed.
+- [ ] Verify that feedback has been provided to the champion.
+- [ ] Propose an optional Bytesize talk about the proposal.
```

Comment:
Give a link to the channel for bytesize proposals

Reply:
We should the bytesize proposal to this repo...
I'll propose that during the next outreach meeting

Reply:
Good idea